ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -420,6 +420,7 @@ private void onNextImpl(AppendEntriesReplyProto reply) {
           grpcServerMetrics.onRequestSuccess(getFollowerId().toString(), reply.getIsHearbeat());
           getLeaderState().onFollowerCommitIndex(getFollower(), reply.getFollowerCommit());
           if (getFollower().updateMatchIndex(reply.getMatchIndex())) {
+            increaseNextIndex(reply.getMatchIndex() + 1, reply.getResult());
```

Comment:
`increaseNextIndex` is used to update nextIndex during snapshot installation. I guess `getFollower().increaseNextIndex()` is what you need instead.

Reply:
Thank you for your reminder. I have replaced it with the correct function.